src/Metal/MTLDevice.cs

```diff
@@ -52,22 +52,34 @@ public static partial class MTLDevice {
 		}
 
 #if MONOMAC
+#if !NET
 		[NoiOS, NoWatch, NoTV]
+#endif
```

Comment:
@spouliot  should we just remove them due to the #if MONOMAC just above? Simpler to review maybe.

Reply:
that case has no value since it's `private`